init.lua

```diff
@@ -379,7 +379,10 @@ require('lspconfig').sumneko_lua.setup {
       diagnostics = {
         globals = { 'vim' },
       },
-      workspace = { library = vim.api.nvim_get_runtime_file('', true) },
+      workspace = {
+        library = vim.api.nvim_get_runtime_file('', true),
+        checkcheckThirdParty = false,
```

Comment:
Looks like there is a typo

```suggestion
        checkThirdParty = false,
```

Reply:
Hah, I was just creating a PR for `checkThirdParty = false`, but then saw this PR. @ngryman Can you please fix the typo per @seantwie03 so this PR can get merged? I'm sure this fix will make a lot of people happier - thanks for submitting it! :)